README.md

```diff
@@ -22,6 +22,18 @@ Add `live_debugger` to your list of dependencies in `mix.exs`:
 
 After you start your application LiveDebugger will be running at a default port `http://localhost:4007`.
 
+### Assets rebuilding
```

Comment:
Tbh I think it should not be optional. It should be a compulsory step when downloading a LiveView debugger. We should also consider code generation I mentioned IRL. I think we'll take care of it after 0.2.0

Reply:
We'll have to also somehow make sure that the user does not have to rebuild assets after each download of the project. Maybe we should keep these files somewhere else